store/v2/root/factory.go

```diff
@@ -32,16 +32,21 @@ const (
 	SCTypeIavlV2 SCType = 1
 )
 
+// app.toml config options
+type Options struct {
+	SSType          SSType               `mapstructure:"ss-type" toml:"ss-type"`
```

Comment:
```suggestion
	SSType          SSType               `mapstructure:"ss-type" toml:"ss-type" comment:"foo bar"`
```

Let's add description for all of those